src/api/hooks/useLogin.ts

```diff
@@ -0,0 +1,31 @@
+import { deleteQueryParams, readLoginQueryParams } from 'Utils/url';
+import useAuthorize from './useAuthorize';
+import { storeClientAccounts } from 'Utils/utility';
+import { useEffect, useMemo } from 'react';
+
+const useLogin = () => {
+    const search = window.location.search;
+    const client_account = readLoginQueryParams();
```

Comment:
```suggestion
    const client_account_params = readLoginQueryParams();
```
just for being more clear